package.json

```diff
@@ -27,7 +27,7 @@
     "walk": "^2.3.9"
   },
   "devDependencies": {
-    "@types/node": "^7.0.29",
+    "@types/node": "7.0.29",
```

Comment:
nice. we'll have to figure out what changed in 7.0.30+ later, but this is a good workaoround